project/ws/app/src/lib/routes/home/components/request-list/create-request-form/create-request-form.component.scss

```diff
@@ -100,6 +100,10 @@
   ::ng-deep.mat-option.mat-selected:not(.mat-option-disabled) {
     color: rgba(0, 0, 0, 0.87) !important;
   }
+
+  ::ng-deep.mat-form-field-flex {
```

Comment:
Please apply this on specific class not on control class

Reply:
added class